src/insexpand.c

```diff
@@ -1946,6 +1946,29 @@ ins_compl_len(void)
     return compl_length;
 }
 
+/*
+ * Retrun TRUE when preinsert is set otherwise FALSE
+ */
+    static int
+ins_compl_has_preinsert(void)
+{
+    return (get_cot_flags() & (COT_PREINSERT | COT_FUZZY)) == COT_PREINSERT;
+}
+
+/*
+ * Checks whether the pre-insert effect cursor position.
```

Comment:
This sentence is confusing and the next sentence is much clearer. I'd suggest just remove this sentence.
```suggestion
```